### PR TITLE
Update - use latest Moesif API package.

### PR DESCRIPTION
**Pull request by praves77**
Pull Request review Checklist:
 - [x] `Follows` guidelines for `public` repo ?
 - [ ] `README` updated, if required ?
 - [x] `Release Version` updated ?
 - [ ] `User Agent` Version updated ?
 - [ ] `Includes` tests conducted and `test result` in the ticket and/or QA doc ?
 - [x] `Does NOT` include code from others?  (Unintended consequence of a bad merge) ?

### CODE REVIEW

File: Moesif.Middleware/Moesif.Middleware.nuspec [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
 			<dependency id="Microsoft.AspNetCore.Http.Extensions" version="2.1.0" />
-			<dependency id="Moesif.Api" version="2.0.5" />
+			<dependency id="Moesif.Api" version="2.0.7" />
 			<dependency id="System.ValueTuple" version="4.5.0" />
-			<dependency id="Newtonsoft.Json" version="9.0.1" />
+			<dependency id="Newtonsoft.Json" version="13.0.2" />
             <dependency id="System.IdentityModel.Tokens.Jwt" version="6.8.0" />
 		</group>
 		<group targetFramework="net45">
 			<dependency id="Microsoft.Owin" version="3.1.0" />
```
Comment (XinyuLuan) [inline Moesif.Middleware/Moesif.Middleware.nuspec:29]: Should we update the version to 4.2.2 too?

Comment (praves77) [reply Moesif.Middleware/Moesif.Middleware.nuspec]: Done

File: Moesif.Middleware/Moesif.Middleware.nuspec [hunk 2]
```diff
@@ -20,16 +20,16 @@
 		<group targetFramework="netstandard2.0">
 			<dependency id="Microsoft.AspNetCore.Http" version="2.1.0" />
```
[the remark this answers is not in the document]
Comment (praves77) [reply Moesif.Middleware/Moesif.Middleware.nuspec]: Done : 1bc0ba6a88c8242093374698df8991f8ad7a329d

**Review by XinyuLuan — APPROVED** [review]
lgtm

**Pull request merged**